### PR TITLE
Current Source Inverter Minimal Implementation added

### PR DESCRIPTION
**Pull request by SabineAuer**
I added a new type of node dynamics `CSIMinimal` that can be the most simple implementation of a grid-feeding inverters: an ideal current source.

FYI: This is still work in progress!

### CODE REVIEW

**Comment by timkittel** [comment]
Are you waiting for a review here? Or are you still working on it?

**Comment by SabineAuer** [comment]
Thanks for asking! I removed the "do not merge" label because I wanted you to have a look at it. :)

**Comment by timkittel** [comment]
OK, I'll have a look tomorrow

**Comment by timkittel** [comment]
Shouldn't that be an ODE with mass? https://github.com/JuliaEnergy/PowerDynBase.jl/blob/4c4d30dc3ad97225b0cabda085edc6964d588535/src/NodeDynamics/CurrentSourceInverterMinimal.jl#L25

File: src/NodeDynamics/CurrentSourceInverterMinimal.jl [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+# Mathematical Representation
+Using `CSIMinimal` for node ``a`` gives:
+```math
+0 = I_{r,a} - \left\|i_a\right\|
+```
+
+"""
+@DynamicNode CSIMinimal(I_r) <: OrdinaryNodeDynamics() begin
```
Comment (timkittel) [inline src/NodeDynamics/CurrentSourceInverterMinimal.jl]: Shouldn't that be an ODE with mass?

Comment (SabineAuer) [reply src/NodeDynamics/CurrentSourceInverterMinimal.jl]: thanks! good point. I changed that.

Comment (timkittel) [reply src/NodeDynamics/CurrentSourceInverterMinimal.jl]: erm, where? there is no commit?

Comment (timkittel) [reply src/NodeDynamics/CurrentSourceInverterMinimal.jl]: @SabineAuer are you going to add the change commit?

Comment (SabineAuer) [reply src/NodeDynamics/CurrentSourceInverterMinimal.jl]: @timkittel: I commited my changes and had some issues with master being merged into this branch, I think.
Now, there is [some coverage issue](https://travis-ci.org/JuliaEnergy/PowerDynBase.jl/jobs/509689162#L644)  again during the travis CI check?

Comment (timkittel) [reply src/NodeDynamics/CurrentSourceInverterMinimal.jl]: No, your `CSIMinimal` test fails (:

(If the tests fail, the coverage reports are not even generated.)

Comment (SabineAuer) [reply src/NodeDynamics/CurrentSourceInverterMinimal.jl]: Thanks! The error message did not help me out. Should have checked the tests. Now they pass :)

**Pull request merged**